tests/rustdoc-ui/intra-doc/weird-syntax.rs

```diff
@@ -117,24 +117,49 @@ pub struct WLinkToCloneWithUnmatchedEscapedCloseParenAndDoubleSpace;
 
 // References
 
-/// The [cln][] link here is going to be unresolved, because `Clone()` gets rejected //~ERROR link
-/// in Markdown for not being URL-shaped enough.
-///
-/// [cln]: Clone() //~ERROR link
+/// The [cln][] link here is going to be unresolved, because `Clone()` gets
+//~^ ERROR link
+/// rejected in Markdown for not being URL-shaped enough.
+/// [cln]: Clone()
+//~^ ERROR link
 pub struct LinkToCloneWithParensInReference;
 
-/// The [cln][] link here is going to be unresolved, because `struct@Clone` gets //~ERROR link
-/// rejected in Markdown for not being URL-shaped enough.
```

Comment:
This comment was wrong.

`struct@Clone` is URL-ish enough to match CommonMark's refdef grammar. The reason it was rejected before was because the `//~ERROR link` hot comment got fed to the markdown parser, and, as a result, caused the refdef to not get parsed there.

By using a below-the-line comment, I can work around that and demonstrate the change correctly.